it/src/test/java/com/linecorp/centraldogma/it/CorsPolicyTest.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.centraldogma.it;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import com.linecorp.armeria.client.WebClient;
+import com.linecorp.armeria.common.AggregatedHttpResponse;
+import com.linecorp.armeria.common.HttpHeaderNames;
+import com.linecorp.armeria.common.QueryParams;
+import com.linecorp.centraldogma.server.CentralDogmaBuilder;
+import com.linecorp.centraldogma.testing.junit.CentralDogmaExtension;
+
+class CorsPolicyTest {
+
+    @RegisterExtension
+    static final CentralDogmaExtension dogma = new CentralDogmaExtension() {
+        @Override
+        protected void configure(CentralDogmaBuilder builder) {
+            builder.corsAllowedOrigins("*");
+        }
+    };
+
+    @Test
+    void updateWriteQuota() throws Exception {
+        WebClient client = dogma.httpClient();
+
+        QueryParams params =
+                QueryParams.builder()
+                           .set(HttpHeaderNames.ORIGIN.toString(), "SomeOrigin")
+                           .set(HttpHeaderNames.ACCESS_CONTROL_REQUEST_METHOD.toString(), "GET")
+                           .build();
+        final AggregatedHttpResponse res = client
+                .options("/api/v1/projects", params).aggregate().join();
```

Comment:
I think the issue is we're setting query parameters instead of the headers.

e.g.
```
final AggregatedHttpResponse res = client.blocking().prepare()
                                         .header(HttpHeaderNames.ORIGIN.toString(), "SomeOrigin")
                                         .header(HttpHeaderNames.ACCESS_CONTROL_REQUEST_METHOD.toString(), "GET")
                                         .options("/api/v1/projects").execute()
```

Reply:
thanks, the fix works